tests/unit/test-util/test.adb.js

```diff
@@ -1311,8 +1311,8 @@ describe('utils/adb', () => {
     });
   });
 
-  describe('test exposed adb API functions', () => {
-    it('test listADBDevices', async () => {
+  describe('exports exposed in util.adb', () => {
+    it('lists connected adb devices', async () => {
```

Comment:
Nit (mainly because this PR would be updated for the changes to the right, and while we are still here let's also do some nits), `it('should export a listADBDevices method', ...)`.